projects/ngx-prx-styleguide/src/assets/styles/_functions.scss

```diff
@@ -0,0 +1,36 @@
+// Color contrast
+@function color-yiq($color, $dark: $yiq-text-dark, $light: $yiq-text-light) {
```

Comment:
Is this function still needed? I'm not seeing it used anywhere.

Reply:
It will come in handy when we need to set text colors on things that will use a theme color as a bg color. Really is there as a tool for when we get into refactoring color usage through out the styleguide.